src/components/breadcrumbs/_breadcrumbs.scss

```
@@ -110,6 +111,10 @@
   margin-bottom: $ouiSizeXS; /* 1 */
 }
 
+.ouiBreadcrumbWall--single {
```

why the additional class and not just

```suggestion
.ouiBreadcrumbWrapper--first.ouiBreadcrumbWrapper--last {
```

---

It gets applied to different elements. You can see where in `src/components/header/__snapshots__/header.test.tsx.snap`. If the background is applied to wrapper, it will get skewed as well, which we don't want for this background.

---

Ah, thanks, yeah the snapshot is helpful in this case. I'm still convinced there are additional simplifications we can make here in the future, but this makes sense for now.